Makefile

```diff
@@ -1,12 +1,10 @@
 .PHONY: clean format test all
 
 clean:
-	pyclean .
-	rm -rf .pdb/ obsolete/ .mypy_cache/ .pytest_cache/ .ruff_cache/ htmlcov/ .coverage
+	pyclean . --debris mypy --erase .pdb/**/* .pdb/ obsolete/**/* obsolete/ --yes
```

Comment:
Watch out that this will *only* clear the mypy folder, but not any other debris (e.g. packaging folders, cache folders, pytest leftovers, etc.).

Of that's what you want and need, fine. Otherwise you need to add any other debris target you require (see the `--help` screen).

I know this is somewhat inconvenient. I've not sat down to come up with a better, more intuitive solution yet. 😟

Reply:
Thank you for the heads up!